src/components/common/ResponsiveVisibility/ResponsiveVisibility.styles.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+
+import { ResponsiveDevices, ResponsiveSize } from '../../../constants/responsive'
+
+export const Wrapper = styled.div<{ values?: string[] }>`
```

Comment:
will it change the tree structure since we have a wrapped div 🤔

Reply:
oh it does not change tree structure, it is added on the exported code only

Reply:
I think the issue is still applied, it might break the layout, for example we have 
```html
<span>
  <span></span>
</span>
```
with this change, it becomes
```html
<span>
  <div style="display: flex">
    <span></span>
  </div>
<span>
```

Reply:
I am not sure it is better or not but what do you think about using javascript to detect screensize instead? 🤔

Reply:
yea, but we need to have wrapper element for visibility

<div>
 <InputComponent />
</div>

how about if I delete display flex?

Reply:
do we really a dom wrapper element? 🤔 I think we can use react component with javascript to detect screensize and hide that component if needed like
```js
const A = () => {
  if (not desktop) {
   return null;
  }
  return component;
}
```

Reply:
ah, this component to support exported project with server side rendering.
on server side, you are not able to get window to get width of window size, so we need to use media query